boards/arm/lpcxpresso55s36/lpcxpresso55s36.dts

```diff
@@ -162,3 +162,8 @@
 	pinctrl-0 = <&pinmux_flexpwm1_pwm0>;
 	pinctrl-names = "default";
 };
+
+&dma0 {
+	status = "okay";
+	dma-channels = <52>;
+};
```

Comment:
isn't there also a dma1?

Reply:
DMA1 will be included in the next push to this branch.